index.html

```diff
@@ -55,5 +59,236 @@ <h4 class="headline-sm-suport">LET'S CONNECT</h4>
     </section>
     <!--headline section ends-->
 
+    <!--Card works starts-->
+    <section class="works">
+        <!-- Card 3 starts -->
+        <li class="works-card">
+          <img src="./assets/Snapshoot Portfolio.png" alt="Recent Work" />
+          <div class="works-card-main">
+            <h2 class="works-card-heading-large">Tonic</h2>
+            <div class="works-card-client">
+              <p class="paragraph">CANOPY</p>
+              <p class="fa-solid fa-circle works-card-client-counter"></p>
+              <p class="works-card-client-role paragraph">Back End Dev</p>
+              <p class="fa-solid fa-circle works-card-client-counter"></p>
+              <p class="works-card-client-year paragraph">2015</p>
+            </div>
+            <p class="paragraph works-card-paragraph">
+              A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="works-card-tags">
+              <li class="works-card-tag">html</li>
+              <!-- <li class="works-card-tag">Ruby on rails</li> -->
+              <li class="works-card-tag">css</li>
+              <li class="works-card-tag">javascript</li>
+            </ul>
+            <button>See Project</button>
+          </div>
+        </li>
+        <!-- Card 3 Ends -->
+      <!-- Card 1 starts -->
+      <ul class="works-cards">
```

Comment:
- [ ] Instead of taking a ul for each li, you can take one ul and deal with each works card as a li, as I mentioned earlier.